File: src/main/webapp/app/js/services/PraiseService.js
angular.module('myApp').service('PraiseService', function ($http, $log) {

    this.savePraise = function(praise){
        return $http({
            method: 'POST',
            data: praise,
            url: '/api/praise'
        });
    }

    this.getAllPraises = function (){
        return $http({
            method: 'GET',
            url: '/praises'
        });
    };
});

File: src/main/java/com/ssoward/service/PraiseService.java
package com.ssoward.service;

import com.ssoward.model.Praise;

import java.util.List;

/**
 * Created by ssoward on 3/1/14.
 */
public interface PraiseService {

    List<Praise> getPraises();

    void savePraise(Praise praise);
}
